packages/clerk-js/src/ui/elements/RadioGroup.tsx

```diff
@@ -18,7 +18,6 @@ const RadioIndicator = forwardRef<HTMLInputElement, { value: string; id: string
       sx={t => ({
         width: 'fit-content',
         marginTop: t.space.$0x5,
-        boxShadow: 'none',
       })}
       type='radio'
```

Comment:
I wouldn't expect this to be needed here.

Reply:
Yeah I have removed that!